Lesson_1.4.js

```diff
@@ -0,0 +1,16 @@
+const fetch = require('node-fetch');
+const url = `http://jsonplaceholder.typicode.com/users/`;
+const arrayOfPromises = [];
+
+for (let i = 1; i <= 10; i++) {
+	arrayOfPromises[i] = new Promise((resolve, reject) => {
+		resolve(
+			fetch(url + i)
```

Comment:
```suggestion
			fetch(url + i).then(response => response.json())
```
ну здесь можно и так (все равно тело функции не слишком огромное) А вот если будут еще then'ы, тогда уже переходи на новые строчки)